.gitignore

```diff
@@ -2,4 +2,7 @@
 /.tox
 /dist
 /tests/source/docs/_build
-/build
\ No newline at end of file
+/build
+*/__pycache__/*
+/.vscode
```

Comment:
Since it's editor-specific, this should probably be in your user's global .gitignore; then you won't have to repeat it for every project.

Reply:
That's a good one. Didn't know I could do that...
The __pycache__ is created by python 3, python 2.7 generate .pyc files in the source. In the next patch I added that one too.